docs/neo-n3/dapp_development/interacting_with_a_node.md

```diff
@@ -133,7 +133,7 @@ necessary methods to interact and make use of such wallets.
 First a wallet needs to be opened.
 
 ```java
-NeoOpenWallet response = neow.openWallet("/path/to/wallet.json", "walletPassword").send();
+NeoOpenWallet response = neow3j.openWallet("/path/to/wallet.json", "walletPassword").send();
```

Comment:
this is about using the same instance as was defined at the start in the document